lte/gateway/c/session_manager/SessionState.h

```diff
@@ -116,6 +116,9 @@ class SessionState {
    */
   void sess_infocopy(struct SessionInfo*);
 
+  SessionFsmState get_state();
+  magma::lte::Fsm_state_FsmState get_smf_state();
```

Comment:
might be good to rename to `get_proto_fsm_state` for clarity I think

Reply:
resolved